Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -984,6 +984,10 @@ - (void)setUpInteraction
     _touchEndDeferringGestureRecognizerForSyntheticTapGestures = adoptNS([[WKDeferringGestureRecognizer alloc] initWithDeferringGestureDelegate:self]);
     [_touchEndDeferringGestureRecognizerForSyntheticTapGestures setName:@"Deferrer for touch end (synthetic tap)"];
 
+    // FIXME: Do I need more?
```

Comment:
Do... you???

Reply:
(OOPS!) As it turned out, I did not! Will remove.

(I forgot to remove that FIXME, which was in reference to _potentially_ requiring another gesture subgraph split by using 2 touchmove deferring gestures.)